calour/tests/test_transforming.py

```diff
@@ -138,12 +139,13 @@ def test_normalize_by_subset_features(self):
         assert_array_almost_equal(newexp.data[:, good_features].sum(axis=1), np.ones([exp.data.shape[0]])*10000)
         self.assertTrue(np.all(newexp.data[:, bad_features] > exp.data[:, bad_features]))
 
+    @skipIf(sys.platform.startswith("win"), "skip this test for Windows")
     def test_subsample_count(self):
         exp = ca.Experiment(data=np.array([[1, 2, 3], [4, 5, 6]]),
                             sample_metadata=pd.DataFrame([['a', 'b', 'c'], ['d', 'e', 'f']]),
                             sparse=False)
         n = 6
-        obs = exp.subsample_count(n)
+        obs = exp.subsample_count(n, random_state=9)
```

Comment:
in analysis.py we call it random_seed instead of random_state. worth changing to random_seed also here?